### PR TITLE
Release 4.1.0

### PR DESCRIPTION
**Pull request by dairbuirabass**
## Checklist:
- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] I have assigned the correct milestone or created one if non-existent.
- [x] I have correctly labeled this pull request.
- [x] I have linked the corresponding issue in this description.
- [x] I have updated the corresponding Jira ticket.
- [x] I have requested a review from at least 2 reviewers
- [x] I have checked the base branch of this pull request
- [x] I have checked my code for any possible security vulnerabilities

### CODE REVIEW

**Review by supercid — CHANGES_REQUESTED** [review]
Check the base

File: nostotagging.php [hunk 1]
```diff
@@ -150,7 +150,7 @@ public function __construct()
         $this->bootstrap = true; // Necessary for Bootstrap CSS initialisation in the UI
         $this->author = 'Nosto';
         $this->need_instance = 1;
-        $this->ps_versions_compliancy = array('min' => '1.5.5.0', 'max' => '2');
+        $this->ps_versions_compliancy = array('min' => '1.5.5.0', 'max' => '8.0.0');
```
Comment (supercid) [inline nostotagging.php:153]: Maybe we should:
```suggestion
        $this->ps_versions_compliancy = array('min' => '1.5.5.0', 'max' < '8.1.0');
```

Comment (supercid) [reply nostotagging.php:153]: or wait, is that even correct?

Comment (dairbuirabass) [reply nostotagging.php:153]: Just in case of a future release? Makes sense.

Comment (dairbuirabass) [reply nostotagging.php:153]: Yeah, that's the versioning now 😄

Comment (supercid) [reply nostotagging.php:153]: I mean, we should support anything lower than 8.1.0, but I'm not sure how this max flag works, the logic might be inverted there

Comment (dairbuirabass) [reply nostotagging.php:153]: It's an array, not a comparison operator

Comment (dairbuirabass) [reply nostotagging.php:153]: https://devdocs.prestashop-project.org/1.7/modules/concepts/module-class/#ps_versions_compliancy

Comment (supercid) [reply nostotagging.php:153]: You're right.
Enough coding for me today 😂

Comment (dairbuirabass) [reply nostotagging.php:153]: 😆 but I will change it to `8.1.0` just to avoid re-releasing only for the sake of module being installable.

**Review by supercid — APPROVED** [review]

**Pull request merged**